ckanext/ontario_theme/templates/internal/package/snippets/resources_list.html

```diff
@@ -43,7 +43,7 @@ <h4 class="empty">{{ _("No date range") }}</h4>
             </ul>
           {% endfor %}
         {% endblock %}
-      <h3>{{ _("Supporting Files") }}</h3>
+      <h2 class="ontario-h2">{{ _("Supporting Files") }}</h2>
```

Comment:
`ontario-h2` can be removed because the h2 tags don't have any overrides

Reply:
done